src/components/Form/Form.jsx

```diff
@@ -15,12 +16,37 @@ export const Form = () => {
     lastDate: null,
   });
 
+  const [value] = useCollection(getAll());
+
+  const isItemDuplicated = (name) => {
+    const regex = /[!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~]/g;
+
+    if (value) {
+      const item = value.docs.filter(
```

Comment:
If we think about how it's being used here, `.filter` is not really being used for its perfectly intended purpose. Usually one uses `.filter` when one wants to end up with a filtered array of items. In this case, we don't really care about using the filtered list of items, we just want to know whether or not one of those, once we normalize it, is equal to the value in our input. So, instead we can use `.some`, which will return a boolean that tells us whether or not the condition in the callback is true.

Then, we can eliminate the `if/else` entirely and just return `value?.docs?.some(...`